lib/waterline/query/dql/create.js

```diff
@@ -215,17 +215,10 @@ function createValues(valuesObject, cb) {
       return after(parentModel.toObject());
     });
 
-
     function after(values) {
```

Comment:
I think it's used on line 209 as well & I'm not super confident in changing it or the test coverage :(